lib/cmap/edges_to_queries.rb

```diff
@@ -22,7 +22,11 @@ def unique_edges
   end
 
   def add_columns_queries
-    unique_edges.map {|e| "alter table #{table_name} add column #{e.destination_vertex} int2;"}
+    unique_edges.map do |e|
+      vertex = e.destination_vertex
+      column_type = vertex.downcase.include?("numeric") ? "float4" : "int2"
```

Comment:
@xta - I think we should make this a bit more generic, so the code works for any column type.  We want `serum_uric_acid_test_result_numeric` and `serum_uric_acid_test_result_numeric_previous` to have the 'real' type.  We can either define an array of all the data types in the cmap gem, so we can write `serum_uric_acid_test_result_real` and `serum_uric_acid_test_result_previous_real` (with a default to int2) or we can make the column type more explicit with something like `serum_uric_acid_test_result_column_type_real`.

The column type is almost always int2, so I think it's important to keep that the default.  Let's come up with a convention that let's the cmap writer create columns that are bigint, date, whatever :smile: